src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/VerifyAsParametersBaseline.generated.txt

```diff
@@ -149,532 +54,608 @@ namespace Microsoft.AspNetCore.Http.Generated
     %GENERATEDCODEATTRIBUTE%
     file static class GeneratedRouteBuilderExtensionsCore
     {
-
-        private static readonly Dictionary<(string, int), (MetadataPopulator, RequestDelegateFactoryFunc)> map = new()
+        private static readonly string[] GetVerb = new[] { Microsoft.AspNetCore.Http.HttpMethods.Get };
+        private static readonly string[] PostVerb = new[] { Microsoft.AspNetCore.Http.HttpMethods.Post };
+        private static readonly string[] PutVerb = new[]  { Microsoft.AspNetCore.Http.HttpMethods.Put };
+        private static readonly string[] DeleteVerb = new[] { Microsoft.AspNetCore.Http.HttpMethods.Delete };
+        private static readonly string[] PatchVerb = new[] { Microsoft.AspNetCore.Http.HttpMethods.Patch };
+
+        [InterceptsLocation(@"TestMapActions.cs", 44, 5)]
+        internal static RouteHandlerBuilder MapGet_445(
```

Comment:
Is the `445` coming from the line and column number? We might want to put a separator in there so line 4 col 45 and line 44 and col 5 don't collide.

Reply:
Correct, I can add a separator. FWIW, the existence of the line numbers/character position in the method name largely serves as a way to differentiate between the different generated interceptors but there is a readability benefit to the separator.

Reply:
The scenario I was thinking of is the case where there are 2 MapGet calls, that have different signatures, with conflicting line/character positions like above.

line 4 col 45
line 44 col 5

If they both generate a method named `MapGet_445` the code isn't going to compile because you will generate 2 methods with the same name and signature.